src/components/Nav/Nav.js

```diff
@@ -1,12 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
 import { CiSearch } from 'react-icons/ci';
 import { BsCart3 } from 'react-icons/bs';
 import { HiOutlineBars3 } from 'react-icons/hi2';
 import { LINK_LIST, LINKBTM_LIST } from './NavData.js';
+import { Category } from './Category.js';
 import './Nav.scss';
 
 const Nav = () => {
+  const [isMouseHover, setIsMouseHover] = useState(false);
+
+  const hoverCategory = () => {
+    setIsMouseHover(true);
+  };
```

Comment:
```suggestion
  const hoverCategory = boolean => {
    setIsMouseHover(boolean);
  };
```

인자 넘겨주는 방식으로 구현하면 하나의 함수로 두가지 동작을 할 수 있겠네요!